data/roadmap/roadmap.yaml

```diff
@@ -48,133 +48,92 @@ categories:
       -
-        status: Done
-        thing: "Repositories page"
+        status: in progress
+        thing: "Monitoring Location Pages Minimal Viable Product (MVP)"
       -
-        status: Done
-        thing: "Save state of hydrograph"
+        status: on deck
+        thing: "Allow two parameters on hydrograph"
       -
-        status: Done
-        thing: "Use [NLDI](https://labs.waterdata.usgs.gov/about-nldi/index.html) to show upstream basin and upstream and downstream monitoring locations"
+        status: on deck
+        thing: "Provide daily statistics for parameters"
       -
-        status: Done
-        thing: "Parameter period of record"
+        status: upcoming
+        thing: "Deliver field and samples measurements"
   -
-    name: "Next-generation networks service and networks pages"
-    status: "in progress"
+    name: "Next-generation WaterAlert"
+    status: "In Progress"
     things:
       -
-        status: done
-        thing: "Network data management"
+        status: in progress
+        thing: "WaterAlert alpha development"
       -
-        status: done
-        thing: "Add network metadata to OGC API"
+        status: on deck
+        thing: "WaterAlert beta development"
       -
-        status: done
-        thing: "Link to network page from monitoring location pages"
+        status: on deck
+        thing: "Communicating the next generation features and roadmap"
       -
-        status: in progress
-        thing: "Put network metadata on Network pages"
+        status: on deck
+        thing: "Transitioning user accounts from legacy to next generation"
       -
-        status: Done
-        thing: "MVP of network pages"
+        status: upcoming
+        thing: "Transition to next generation WaterAlert"
       -
-        status: Done
-        thing: "MVP network metadata management and services"
+        status: upcoming
+        thing: "Decommissioning legacy WaterAlert"
   -
-    name: "Stakeholder Outreach"
+    name: "Stakeholder Outreach and Discovery"
     status: "In Progress"
     things:
       -
-        status: "In Progress"
-        thing: "Water quality outreach and listening sessions"
+        status: done
+        thing: "Discovery on groundwater censored data"
       -
-        status: "In Progress"
-        thing: "NWIS Annual Survey, release for public on July 20, 2020"
+        status: in progress
+        thing: "NWIS Annual Survey"
       -
-        status: done
-        thing: "Make a publicly accessible roadmap"
+        status: in progress
+        thing: "Discovery on water quality tabular data"
       -
-        status: Done
-        thing: "NWISweb user research (18F Phase I)"
+        status: on deck
+        thing: "Discovery on modernizing WaterQualityWatch, WaterWatch, GroundWaterWatch"
       -
-        status: "In Progress"
-        thing: "WDFN user testing (18F Phase II)"
+        status: upcoming
+        thing: "Discovery on delivering multiple sites simultaneously"
```

Comment:
Also in discovery ( on roadmap): Services and Documentation, and Reference Lists (also redoing help.waterdata)

Reply:
Added an upcoming thing for that other discovery